app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
```

Comment:
Please read the Devise documentation and decide what modules you are going to use in your project, then either uncomment or delete them.
We don't keep code because we **might** use it in future.
https://www.wikiwand.com/en/You_aren%27t_gonna_need_it